### PR TITLE
Palette smartphone adjustments

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

As per feedback from the command palette PR, this should make the feature a fine experience on devices as small as the iPhone 5 (if you for some reason hook up a physical keyboard to one).
Also touched up some palette logic and typing and added property `isMobile` to palette usage events reported to PostHog – just so we know how many power users are commanding on the go.

iPhone 5 Horizontal

iPhone 5 Portrait

### CODE REVIEW

File: frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts [hunk 1]
```diff
@@ -198,6 +202,7 @@ export const commandPaletteLogic = kea<
             // eslint-disable-next-line @typescript-eslint/no-unused-vars
             const { resolver, ...cleanedCommand } = cleanedResult.source
             cleanedResult.source = cleanedCommand
+            cleanedResult.isMobile = isMobile()
```
Comment (mariusandra) [inline frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts:205]: This is working great now on all resolutions I tried! Thank you! Again, I really appreciate you taking the time to fix this. 

The only comment I have is about this line. Where is the `isMobile` attribute on the result used?

Comment (Twixes) [reply frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts:205]: `cleanedResult` is only used as 'palette command executed' event properties.

Comment (mariusandra) [reply frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts:205]: doh! should have checked one line below.

**Review by mariusandra — APPROVED** [review]
Good for me!

**Pull request merged**